php/7.4/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM php:7.4-fpm-alpine
+
+RUN apk upgrade --update \
+    && apk --update add --no-cache --virtual .phpize-deps $PHPIZE_DEPS imagemagick-dev libtool \
```

Comment:
Could you put a bash into the container in a separate apk add command while also merging imagemagick from line 12 into it?

Reply:
Please take a look if this works.
```suggestion
    && apk --update add --no-cache --virtual .phpize-deps $PHPIZE_DEPS imagemagick-dev libtool \
    && apk --update add --no-cache .imagick-runtime-deps imagemagick bash \
```

Reply:
Also I assume line 12 will change to
```
&& apk add --no-cache --virtual .imagick-runtime-deps \
```

Am I assumed it correctly?

Reply:
It can be deleted

Reply:
Alright, I'll check it

Reply:
Great! It worked! I'll push my modifications for a re-check.